src/bmark/bmark_solvers.jl

```diff
@@ -1,4 +1,7 @@
-export bmark_solvers, profile_solvers, bmark_and_profile
+export bmark_solvers, profile_solvers, profile_solvers_by_time,
+       profile_solvers_by_iterations, profile_solvers_by_evaluations,
+       bmark_and_profile, bmark_and_profile_by_time,
+       bmark_and_profile_by_iterations, bmark_and_profile_by_evaluations
```

Comment:
The user could pass a function to be used in the profile, e.g., `stats -> stats[:eval] + stats[:time]`. The default could be the sum of the objective + gradient evals, or something like that.